package.json

```diff
@@ -27,9 +27,9 @@
 		"cryptocurrency"
 	],
 	"dependencies": {
+    "isomorphic-fetch": "^2.2.1",
```

Comment:
Weird, I added in the correct whitespace, but this comment isn't being shown as outdated. It should have been fixed in my last commit.